client/css/style.css

```diff
@@ -568,17 +568,11 @@ kbd {
 	color: #49505a;
 }
 
-#sidebar .chan.active .badge,
-#sidebar .badge:empty {
-	opacity: 0;
-}
-
 #sidebar .close {
 	border-radius: 3px;
 	width: 18px;
 	height: 18px;
-	visibility: hidden;
-	opacity: 0;
+	display: none;
 	transition: opacity 0.2s, background-color 0.2s;
```

Comment:
No opacity/visibility transition on click anymore (switching from opacity/visibility to display in order to hide/show), but there is still an opacity/bg-color transition on hover as opposed to the badge.